pkg/apis/submariner/v1alpha1/servicediscovery_types.go

```diff
@@ -35,6 +35,7 @@ type ServiceDiscoverySpec struct {
 	Debug                    bool   `json:"debug"`
 	ClusterID                string `json:"clusterID"`
 	Namespace                string `json:"namespace"`
+	GlobalnetEnabled         bool   `json:"globalnetEnabled,omitempty"`
```

Comment:
I think omitempty needs to be a pointer to bool.

Reply:
> I think omitempty needs to be a pointer to bool.

We're not using pointer for other bool `omitempty`